Controllers/ViewController.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  AdoptPet.swift
+//  Paws4Me
+//
+//  Created by Kirsty-lee Walker 20/02/2022
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+
+    @IBOutlet weak var petTable: UITableView!
+    // Creates an pet object for users to reference the data
+    var adoptPetObject:AdoptPet?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.title="Adoption Animals"
```

Comment:
You can adjust the spacing here to make this a little bit easier to read:

`self.title = "Adoption Animals"`

Reply:
Noted. 👍🏻